### ISSUE TITLE
Accessibility and color contrast in item-slideshow. 

### ISSUE THREAD

**Issue by christinach** [pulibrary/pomegranate#353]
https://dpul-staging.princeton.edu/slavic

![item-slideshow_activeandhover](https://user-images.githubusercontent.com/9905193/39327657-a1a463bc-4966-11e8-82a9-003e0e0cae4d.png)

![item-slideshow_active](https://user-images.githubusercontent.com/9905193/39327681-b44cbf6e-4966-11e8-8e71-ee025b086626.png)

![audits_accessibility](https://user-images.githubusercontent.com/9905193/39327725-d4aaa83e-4966-11e8-98f8-ffbd382faa9c.png)

**Comment by christinach** [comment]
![axe_tool](https://user-images.githubusercontent.com/9905193/39330399-82832640-496f-11e8-9565-fd77721cfd6c.png)

**Comment by axamei** [comment]
The color contrast is not sufficient. It should be at least 4.5:1. Another issue is that this is not a typical pattern for slideshows. Generally, slideshow patterns include arrows or other indicators that make it known to the user that there is a series of images they can navigate.

**Comment by christinach** [comment]
@axamei I changed it to a light grey. Regarding the indicators I would need more feedback @sdellis 

![lighrgrey](https://user-images.githubusercontent.com/9905193/39331966-065d2d04-4974-11e8-9246-2357bad7b2f7.png)

**Comment by christinach** [comment]
![screen shot 2018-05-02 at 4 41 32 pm](https://user-images.githubusercontent.com/9905193/39548214-e1387160-4e27-11e8-85e7-91b1d7f5e250.png)
